### PR TITLE
[ISSUE #7437] Add the CRC check of commitlog 

### LINKED ISSUE #7437: [Bug] A bug in the CRC check of commitlog
### Before Creating the Bug Report

- [X] I found a bug, not just asking a question, which should be created in [GitHub Discussions](https://github.com/apache/rocketmq/discussions).

- [X] I have searched the [GitHub Issues](https://github.com/apache/rocketmq/issues) and [GitHub Discussions](https://github.com/apache/rocketmq/discussions)  of this repository and believe that this is not a duplicate.

- [X] I have confirmed that this bug belongs to the current repository, not other repositories of RocketMQ.

### Runtime platform environment

Linux

### RocketMQ version

5.1.5

### JDK Version

JDK11

### Describe the Bug

A shutdown occurs when the message is not completely written, causing all characters at the end of the message to be unwritten.
image

recover uses magicCode and bodyCrc to verify whether the writing is successful, but properties do not participate in the verification and belong to the end of the entire message.

Therefore, it cannot be verified that the data of this message is wrong. If you continue to use this error message, unpredictable consequences will occur, such as blocking timerMessage distribution.

### Steps to Reproduce

When there is an abnormal downtime, the body of the last message is written successfully, but the properties are not written successfully.

### What Did You Expect to See?

During recover, data abnormalities in any part of the entire message can be verified.

### What Did You See Instead?

Exceptions other than the body field cannot be found during recover.

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by guyinyou**
### Which Issue(s) This PR Fixes

Fixes #7437

### CODE REVIEW

File: store/src/main/java/org/apache/rocketmq/store/CommitLog.java [hunk 1]
```diff
@@ -61,7 +61,6 @@
 import org.apache.rocketmq.store.logfile.MappedFile;
 import org.apache.rocketmq.store.util.LibC;
 import org.rocksdb.RocksDBException;
-
```
Comment (joeCarf) [inline store/src/main/java/org/apache/rocketmq/store/CommitLog.java:64]: maybe no need to reformat the import in this file and `MessageConfig.java`

**Review by RongtongJin — COMMENTED** [review]
Add some Uts

File: store/src/main/java/org/apache/rocketmq/store/CommitLog.java (hunk 1 above)
Comment (guyinyou) [reply store/src/main/java/org/apache/rocketmq/store/CommitLog.java]: done

**Comment by guyinyou** [comment]
> Add some UTs

done

**Review by joeCarf — APPROVED** [review]
lgtm

**Review by RongtongJin — APPROVED** [review]

**Review by guyinyou — COMMENTED** [review]
lgtm

**Pull request merged**